panic: runtime error: invalid memory address or nil pointer dereference

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0xa86a6f]

goroutine 759 [running]:
github.com/zalando/skipper/dataclients/kubernetes.readDefaultFilters({0x7ffffb1f217a, 0x1b})
	/workspace/dataclients/kubernetes/defaultfilters.go:34 +0x1af
github.com/zalando/skipper/dataclients/kubernetes.(*Client).fetchDefaultFilterConfigs(0xc00025c000)
	/workspace/dataclients/kubernetes/kube.go:512 +0x2e
github.com/zalando/skipper/dataclients/kubernetes.(*Client).loadAndConvert(0xc00013c230)
	/workspace/dataclients/kubernetes/kube.go:344 +0x46
github.com/zalando/skipper/dataclients/kubernetes.(*Client).LoadUpdate(0xc00013c230)
	/workspace/dataclients/kubernetes/kube.go:452 +0x67
github.com/zalando/skipper/routing.receiveFromClient({0x1120e88, 0xc00013c230}, {0xc00026d110, 0x0, 0xb2d05e00, {0xc00007f130, 0x1, 0x1}, {0xc0001bf680, 0x14, ...}, ...}, ...)
	/workspace/routing/datasource.go:90 +0xab
created by github.com/zalando/skipper/routing.receiveRouteDefs
	/workspace/routing/datasource.go:175 +0x2c5
```

It looks like `DirEntry.Info()` err is not checked
https://github.com/zalando/skipper/blob/7f4a3e9c5a5dddf13b2ceb7036448e2bf1a1ba77/dataclients/kubernetes/defaultfilters.go#L33-L37
and info is nil which fails on the `info.Size()`

--- a/dataclients/kubernetes/defaultfilters.go
+++ b/dataclients/kubernetes/defaultfilters.go
@@ -31,7 +31,7 @@ func readDefaultFilters(dir string) (defaultFilters, error) {
 	for _, f := range files {
 		r := strings.Split(f.Name(), ".") // format: {service}.{namespace}
 		info, err := f.Info()
-		if len(r) != 2 || !(f.Type().IsRegular() || f.Type()&os.ModeSymlink != 0) || info.Size() > maxFileSize {
+		if err != nil || len(r) != 2 || !(f.Type().IsRegular() || f.Type()&os.ModeSymlink != 0) || info.Size() > maxFileSize {
 			log.WithError(err).WithField("file", f.Name()).Debug("incompatible file")
 			continue
 		}

--- a/dataclients/kubernetes/kube_test.go
+++ b/dataclients/kubernetes/kube_test.go
@@ -3229,17 +3229,7 @@ func TestSkipperDefaultFilters(t *testing.T) {
 			t.Error(err)
 		}
 
-		dc, err := New(Options{
-			KubernetesURL:     api.server.URL,
-			DefaultFiltersDir: defaultFiltersDir,
-		})
-		if err != nil {
-			t.Error(err)
-		}
-
-		defer dc.Close()
-
-		df, err := readDefaultFilters(dc.defaultFiltersDir)
+		df, err := readDefaultFilters(defaultFiltersDir)
 
 		if err != nil || len(df) != 0 {
 			t.Error("should return empty slice", err, df)

